src/assets/i18n/ar.json5

```diff
@@ -877,9 +877,410 @@
   // "admin.access-control.groups.form.return": "Return to groups",
   // TODO New key - Add a translation
   "admin.access-control.groups.form.return": "Return to groups",
+
+  //"admin.reports.collections.title": "Collection Filter Report",
```

Comment:
I've noticed that most of this PR is simply updates to these language packs.  You don't need to make these changes to every single language pack.  You can just update `en.json5` and that's OK (though you are welcome to edit others if you want).  I just wanted to point this out though because it makes this PR look a lot larger than it really is & that might intimidate other code reviewers.